lib/basedriver/helpers.js

```diff
@@ -110,7 +110,7 @@ async function configureApp (app, supportedAppExtensions) {
       if (headers['content-type']) {
         logger.debug(`Content-Type: ${headers['content-type']}`);
         // the filetype may not be obvious for certain urls, so check the mime type too
-        if (ZIP_MIME_TYPES.includes(headers['content-type'])) {
+        if (ZIP_MIME_TYPES.some(mimeType => new RegExp(`\\b${_.escapeRegExp(mimeType)}\\b`).test(headers['content-type']))) {
```

Comment:
we could even add an option to ignore case here to cover the craziest content types :)